rdm/util.py

```diff
@@ -1,5 +1,12 @@
 import sys
 
+import yaml
+
+
+def get_project_settings(system_yml_path):
```

Comment:
Perhaps we can generalize this to just `load_yaml`?

Reply:
Hmm... I had imagined this function serving a more domain-specific purpose, but at the end of the day it's just a `load_yaml` function. I'm fine with generalizing it.